scalableminds/webknossos
# Refactor Backend: merge Annotation and AnnotationSQL

### Steps to test:
- create / delete / finish explorative + task annotations
- download compound tasktype/project etc
- reset task to base
- check list views
- upload NMLs
- etc

### Dev deployment for testing
 - https://refactorbackendannotation.webknossos.xyz/

### Issues:
- contributes to #2521

------
